README.md

```diff
@@ -125,6 +125,28 @@ dashboardTemplates:
 
 Note that you can define more than one dashboard under the `dashboardTemplates` field. 
 
+#### Entity expiration 
+
+By default, entities are automatically deleted if we reach 8 days without receiving any telemetry from them. 
+If this doesn't suit your needs you may also specify your desired `expirationPolicy` and `expirationTime` under the `expiration` section of the main `definition.yml` file:  
+
+```yaml
+expiration:
+  # The entity deletion policy. Defaults to AUTOMATIC
+  expirationPolicy: AUTOMATIC
+  # The amount of time without receiving telemetry before an entity is deleted. Defaults to EIGHT_DAYS
+  expirationTime: EIGHT_DAYS
+```
+
+`expirationPolicy` accepts the values `AUTOMATIC` (default) and `MANUAL`. 
```

Comment:
Do we even want to expose the manual one? There's no way to manually delete entities from the outside

Reply:
There are some entities that use it, like `nr1-workload`,  so we'd have to store that information here. Consulting with Miguel this is something that it seems like do need to support, but it wouldn't make sense for synthesized entities probably, so an option is to leave it in the schema (so that we can add it on the manual ones), and don't document it + add a validation in the emitter that if the synthesis section is not null, the expirationPolicy cant be manual, wdyt?

Reply:
Can we simplify this into 

```
expirationTime (EIGHT_DAYS, QUARTERLY, MANUAL)
```

Having one less parameter makes it easy to understand it. And we can map into whatever we need internally.
Sounds it better? or is just worse to do it this way?

Reply:
That's a good idea, it's easy to understand